src/commands/deployWorkspaceProject/DeployWorkspaceProjectSaveSettingsStep.ts

```diff
@@ -37,7 +37,9 @@ export class DeployWorkspaceProjectSaveSettingsStep extends ExecuteActivityOutpu
         return !!context.shouldSaveDeploySettings;
     }
 
-    protected createSuccessOutput(): ExecuteActivityOutput {
+    protected createSuccessOutput(context: DeployWorkspaceProjectContext): ExecuteActivityOutput {
+        context.telemetry.properties.didSaveSettings = 'true';
```

Comment:
I think I would prefer this to be in `ExecuteActivityOutputStepBase` as a generic property. Maybe something like
`properties.createOutput = 'success'|'fail'`. I think that this information could be useful for all of these type of steps.

Reply:
We'll have to discuss this one offline, I'm not entirely sure how this property would get mapped back to the step it is associated with